src/vs/workbench/contrib/searchEditor/browser/searchEditorSerialization.ts

```diff
@@ -208,7 +208,7 @@ export const extractSearchQueryFromLines = (lines: string[]): SearchConfiguratio
 };
 
 export const serializeSearchResultForEditor =
-	(searchResult: SearchResult, rawIncludePattern: string, rawExcludePattern: string, contextLines: number, labelFormatter: (x: URI) => string): { matchRanges: Range[], text: string, config: Partial<SearchConfiguration> } => {
+	(searchResult: SearchResult, rawIncludePattern: string, rawExcludePattern: string, contextLines: number, labelFormatter: (x: URI) => string, sortOrder: SearchSortOrder = SearchSortOrder.Default): { matchRanges: Range[], text: string, config: Partial<SearchConfiguration> } => {
```

Comment:
I'd remove the default parameter on `sortOrder`, instead it should be either required, or optional so the fallback occurs at the existing `searchMatchComparer`, that way if it's ever changed in the future we just make one change.

Reply:
I made it required since we provide it in all the place.